### PR TITLE
Open sharepoint file in teams file Viewer C#

### PR DESCRIPTION
**Pull request by NikitaJain-MSFT**
Bot which shares SharePoint file which can be viewed in Teams file viewer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +66,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 |9| Universal bots  | Teams catering bot demonstrating how to use Universal bots in Teams | [View](samples/bot-teams-catering/csharp) |  | |
 |10| Sequential workflow adaptive cards | Demonstrating on how to implement sequential flow, user specific view and upto date adaptive cards in bot. | [View][sequential#workflow#csharp] | [View][sequential#workflow#js]
 |11| Channel messages with RSC permissions | Demonstrating on how a bot can receive all channel messages with RSC without @mention. | [View][messageswithrsc#csharp] |[View][messageswithrsc#js]
+|12| Bot with SharePoint file viewer| This sample demos a bot which uploads file to SharePoint site and same files can be viewed in Teams file viewer. | [View][botwithsharepointfileviewer#csharp] |
```
Comment (v-chetsh) [inline README.md:69]: "Bot with SharePoint file to view in Teams file viewer"

Comment (v-chetsh) [inline README.md:69]: This sample demos a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer.

File: samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/AppManifest/manifest.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Bot SharePoint file viewer",
+    "full": "Bot SharePoint file viewer"
+  },
+  "description": {
+    "short": "Bot SharePoint file viewer",
+    "full": "This sample demos a bot which uploads file to SharePoint site and same files can be viewed in Teams file viewer"
```
Comment (v-chetsh) [inline samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/AppManifest/manifest.json:19]: This sample demos a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer.

File: samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/Controllers/SharepointFileUploadController.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        public readonly IConfiguration _configuration;
+        public SharePointFileUploadController(
+            ConcurrentDictionary<string, TokenState> token,IConfiguration configuration)
+        {
+            _token = token;
+            _configuration = configuration;
+        }
+
+        /// <summary>
+        /// This enpoint is called to save the updated changes for particular product based on id.
```
Comment (v-chetsh) [inline samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/Controllers/SharepointFileUploadController.cs:27]: endpoint

File: samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/SimpleGraphClient.cs [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            {
+                var drive = await graphClient.Sites[site.Id].Drives
+                                    .Request()
+                                    .GetAsync();
+                if (drive != null)
+                {
+                    await graphClient.Sites[site.Id].Drives[drive.CurrentPage[0].Id].Root.ItemWithPath(fileName).Content
+                                            .Request()
+                                            .PutAsync<DriveItem>(stream);
+                }
```
Comment (v-chetsh) [inline samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/SimpleGraphClient.cs:87]: Looks like it is not aligned properly.

File: samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/helper/SharePointFileHelper.cs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace BotWithSharePointFileViewer.helper
+{
+    public class SharePointFileHelper
+    {
+        // Get sharepoint file list.
```
Comment (v-chetsh) [inline samples/bot-sharepoint-file-viewer/csharp/BotWithSharePointFileViewer/helper/SharePointFileHelper.cs:15]: SharePoint

File: samples/bot-sharepoint-file-viewer/csharp/README.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
+﻿﻿﻿Using this C# sample, a bot can upload file to SharePoint site and same files can be viewed in Teams file viewer.
```
Comment (v-chetsh) [inline samples/bot-sharepoint-file-viewer/csharp/README.md:1]: a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer

**Review by v-chetsh — CHANGES_REQUESTED** [review]
Reviewed and provided my feedback.

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving the PR.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**